contrib/kml_export.lua

```diff
@@ -230,23 +229,16 @@ function addDuplicateIndex( index, filename )
 end
 
 local function create_kml_file(storage, image_table, extra_data)
-  if not df.check_if_bin_exists("mkdir") then
-    dt.print_error(_("mkdir not found"))
-    return
-  end
   if not df.check_if_bin_exists("convert") then
```

Comment:
At first glance I suspect check_if_bin_exists('convert') will fail on windows. I have always had to put a file chooser in the preferences so the user can point to the program and then perform a dt.preferences.write('executable_paths', 'someprogram', dir) for check_if_bin_exists('someprogram') to work on windows. It seems as though when installing programs on windows they are not automatically visible to dt in the way they are on linux machines. 

@wpferguson please correct me if I'm wrong here as I'm not 100% certain. If there is a way around this I would love to know, as I'm currently trying to access dts exiv2 but can't figure out how.

Reply:
@BzKevin  sounds reasonable. I'll look into that and a preference.

Reply:
I pushed a new version